presentation/src/main/java/com/dcs/presentation/common/PercentageCircleView.kt

```diff
@@ -0,0 +1,102 @@
+package com.dcs.presentation.common
+
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.geometry.Size
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Preview(name = "PercentageCircleView", showBackground = true)
+@Composable
+fun PercentageCircleView(
```

Comment:
```suggestion
fun CircularProgressBarWithPercentage
```

기능적인 것 보단 외형적 생김새를 봤을 때에는 원형 프로그래스바라고 명명하는게 좋지 않을까요? 그리고 compose로 함수 선언할 때 `View` 를 안 붙이는게 일반적이더라구요. 기존에는 View를 확장해서 사용하다보니 XXXView를 썼는데 compose로 와서는 안 쓰는 추세입니다.

Reply:
적용 완료